src/lib/utils/preconditions/PreconditionContainerArray.ts

```diff
@@ -151,10 +152,10 @@ export class PreconditionContainerArray implements IPreconditionContainer {
 	 * @param message The message that ran this precondition.
 	 * @param command The command the message invoked.
 	 */
-	public run(message: Message, command: Command): PreconditionContainerReturn {
+	public run(message: Message, command: Command, context: PreconditionContext): PreconditionContainerReturn {
```

Comment:
Isn't this added required parameter a breaking change? Please document it if so.